### DISCUSSION TITLE
nested arrays are not supported

### DISCUSSION THREAD

**Discussion by subhadip1203** [FerretDB/FerretDB#4458] [General]
As of the current FerretDB  release: nested arrays are not supported

in the future is FerretDB going to implement this?  is there any future plan?

**Comment by AlekSi** [comment]
We have already implemented them in the next major version. I should be out in a few months.

**Reply by subhadip1203** [reply]
It's great to hear that. I appreciate your answer.
what an awsome project.

**Reply by AlekSi** [reply]
It will be out in a few weeks: #4578

**Reply by subhadip1203** [reply]
Any update on the "nested arrays are not supported". on documentation, it is still showing that nested arrays are not supported

**Reply by AlekSi** [reply]
v2 should be released in a few weeks (for real this time 😄)
